pkg/local_workflows/report_analytics_workflow.go

```diff
@@ -94,16 +99,39 @@ func reportAnalyticsEntrypoint(invocationCtx workflow.InvocationContext, inputDa
 	return nil, nil
 }
 
-func callEndpoint(invocationCtx workflow.InvocationContext, input workflow.Data, url string) error {
+func normalizeInput(input []byte) []byte {
+	var scanDoneEvent json_schemas.ScanDoneEvent
+	err := json.Unmarshal(input, &scanDoneEvent)
+	if err != nil {
+		return input
+	}
+
+	durationMs, err := strconv.ParseInt(scanDoneEvent.Data.Attributes.DurationMs, 10, 64)
+	if err != nil {
+		return input
+	}
+
+	if durationMs > 1000*3600*12 || durationMs < 0 {
```

Comment:
If the duration is longer than 12 hours, we make it look like 0? That seems wrong

Reply:
I think it's better to capture what the client actually sees. If you hide it, you can't troubleshoot it. If it messes up reports, that can be filtered on the reporting side

Reply:
or some other way to indicate the error

Reply:
How and where is this used? If we surface the error, can we handle it explicitly up the stack?

If it goes into an analytics request, maybe the better choice is to set the duration to null, or reject the request.

If we're getting durations > 12h, isn't that a bug in the caller? I don't know that hiding that is helping us address the root cause.